sql/moz-fx-data-shared-prod/org_mozilla_firefox_derived/experiment_events_live_v1/init.sql

```diff
@@ -0,0 +1,66 @@
+-- This and the following materialized view need to be kept in sync:
+-- - org_mozilla_firefox_beta_derived.experiment_events_live_v1
+-- - org_mozilla_fenix_derived.experiment_events_live_v1
+-- - org_mozilla_firefox_derived.experiment_events_live_v1
+CREATE MATERIALIZED VIEW org_mozilla_firefox_derived.experiment_events_live_v1
+OPTIONS
+  (enable_refresh = TRUE, refresh_interval_minutes = 5)
+AS
+WITH fenix_all_events AS (
+  SELECT
+    submission_timestamp,
+    events
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_firefox_live.events_v1`
+),
+fenix AS (
+  SELECT
+    submission_timestamp AS `timestamp`,
+    event.category AS `type`,
+    -- using UNNEST twice on the same field doesn't work in materialized views
+    REGEXP_EXTRACT(TO_JSON_STRING(event.extra), '.*{"key":"branch","value":"([^"]*)"}.*') AS branch,
+    REGEXP_EXTRACT(
+      TO_JSON_STRING(event.extra),
+      '.*{"key":"experiment","value":"([^"]*)"}.*'
+    ) AS experiment,
+    event.name AS event_method
```

Comment:
The materialized views showed some unexpected behaviour when `UNNEST`ing and `CROSS JOINING` the same field twice in this query. The `branch` value ended up being set to the `experiment`. When running the query as is (without creating the materialized view) in the console `experiment` and `branch` values showed up correctly. 

I was able to re-produce this issue with a simpler query: 
https://console.cloud.google.com/bigquery?sq=1061347137128:a129bb55dadc4a1f88fcb48a26276803

I'll open a bug for this in the BigQuery issue tracker. For now this workaround does also return the correct results.